tests/positive/passwd/users.go

```diff
@@ -160,6 +161,96 @@ ENCRYPT_METHOD SHA512
 	}
 }
 
+// DeletePasswdUsers verifies that the user(s) can be removed
+// from a given OS distro.
+func DeletePasswdUsers() types.Test {
```

Comment:
Let's put a shouldExist: true on one of the users from `AddPasswdUsers`, also.

And we'll need a test for removing groups.